provisioning/transport/http/src/Generated/DeviceProvisioningServiceRuntimeClient.cs

```diff
@@ -0,0 +1,247 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Do we keep the "Generated" folder?

Reply:
This code was generated initially then I've fixed all FXCop issues and formatted using our coding standard.
Unless there's a good reason, we could keep this as-is in case we need to pick-up & regen when the service changes in the future.